Files/Views/LayoutModes/ColumnViewBase.xaml

```diff
@@ -1171,167 +1260,247 @@
+
+                    <ListView.GroupStyle>
+                        <GroupStyle>
+                            <GroupStyle.HeaderTemplate>
+                                <DataTemplate x:DataType="helpers:IGroupedCollectionHeader">
+                                    <StackPanel
+                                        Margin="5,4,0,4"
+                                        AutomationProperties.Name="{x:Bind Model.Text}"
+                                        Orientation="Vertical">
+                                        <StackPanel Orientation="Horizontal" Spacing="5">
+                                            <TextBlock
+                                                VerticalAlignment="Bottom"
+                                                FontSize="16"
+                                                Style="{StaticResource SubtitleTextBlockStyle}"
+                                                Text="{x:Bind Model.Text, Mode=OneWay}" />
+                                            <TextBlock
+                                                VerticalAlignment="Bottom"
+                                                FontSize="14"
+                                                Foreground="{StaticResource SystemControlPageTextBaseMediumBrush}"
+                                                Style="{StaticResource SubheaderTextBlockStyle}"
+                                                Text="{x:Bind Model.Subtext, Mode=OneWay}" />
+                                        </StackPanel>
+                                        <TextBlock
+                                            Foreground="{StaticResource SystemControlPageTextBaseMediumBrush}"
+                                            Style="{StaticResource BodyTextBlockStyle}"
+                                            Text="{x:Bind Model.CountText, Mode=OneWay}" />
+                                    </StackPanel>
+                                </DataTemplate>
+                            </GroupStyle.HeaderTemplate>
+                        </GroupStyle>
+                    </ListView.GroupStyle>
+                    <ListView.ItemsPanel>
+                        <ItemsPanelTemplate>
+                            <ItemsStackPanel AreStickyGroupHeadersEnabled="True" Orientation="Vertical" />
+                        </ItemsPanelTemplate>
+                    </ListView.ItemsPanel>
+                </ListView>
+            </SemanticZoom.ZoomedInView>
+            <SemanticZoom.ZoomedOutView>
+                <ListView
+                    HorizontalAlignment="Stretch"
+                    IsItemClickEnabled="True"
+                    ItemsSource="{x:Bind CollectionViewSource.View.CollectionGroups, Mode=OneWay}"
+                    SelectionMode="None">
+                    <ListView.ItemTemplate>
+                        <DataTemplate x:DataType="ICollectionViewGroup">
+                            <StackPanel
+                                Margin="0,5,0,5"
```

Comment:
Is the odd margin on purpose?

Reply:
No, just a bad habit xD

Reply:
There are a couple other values like this as well, can you go through all of the ones in this PR?